Readme.md

```diff
@@ -38,3 +38,25 @@ curl --include \
    --header "Sec-WebSocket-Version: 13" \
    http://localhost:8888/ws
 ```
+
+## Roadmap
+
+- [ ] Framing
+  - [x] Text Messages
+  - [ ] Binary Messages
```

Comment:
What is the issue with binary messages?

Reply:
Implemented it here. https://github.com/status-im/nim-ws/pull/9/commits/9e98713be0952a129ac5403017176cee18d9cf11
Just had to expose the opcode in `recv`

Reply:
ok. thx